docs/pages/admin-guides/infrastructure-as-code/terraform-provider.mdx

```diff
@@ -17,14 +17,15 @@ is executed. You must pick the correct guide for your setup:
 
 | Guide                                                                                                               | Use-case                                                                                                                                                                                                         | How it works                                                                                                                           |
 |---------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------|
-| [Run the Teleport Terraform provider locally](./terraform-provider/local.mdx)                                       | This is the best way to get started with the Teleport terraform provider, write some initial Terraform code and get familiar with IaC.                                                                           | You're using you local credentials to create a temporary bot, obtain short-lived credentials, and store them in environment variables. |
+| [Run the Teleport Terraform provider locally](./terraform-provider/local.mdx)                                       | This is the best way to get started with the Teleport terraform provider, write some initial Terraform code and get familiar with IaC.                                                                           | You're using local credentials to create a temporary bot, obtain short-lived credentials, and store them in environment variables. |
```

Comment:
```suggestion
| [Run the Teleport Terraform provider locally](./terraform-provider/local.mdx)                                       | You are getting started with the Teleport Terraform provider and managing Teleport resources with IaC.                                                                           | You use local credentials to create a temporary bot, obtain short-lived credentials, and store them in environment variables. |
```

Cleaning up the description so that the language in each cell parallels the language of its neighbors in a given column.